File: app/src/main/java/com/example/massa/luxvilla/LuxVilla.java
package com.example.massa.luxvilla;

import android.app.Application;

import com.google.firebase.database.FirebaseDatabase;

/**
 * Created by massa on 05/12/2016.
 */

public class LuxVilla extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        FirebaseDatabase.getInstance().setPersistenceEnabled(true);

    }
}

File: app/src/main/java/com/example/massa/luxvilla/Application/LuxVilla.kt
package com.example.massa.luxvilla.Application

import android.app.Application
import com.google.firebase.database.FirebaseDatabase

/**
 * Created by massa on 27/10/2017.
 */
class LuxVilla : Application() {

    override fun onCreate() {
        super.onCreate()

        FirebaseDatabase.getInstance().setPersistenceEnabled(true)

    }
}
